Segunda_parte/ejercicio15.c

```diff
@@ -0,0 +1,41 @@
+#include <stdio.h>
+#include <string.h>
+
+/*
+Julius Caesar protegia su información confidencial encriptandola usando 
+un “cipher” (cifrador?). El cipher de Caesar, rota cada letra del abecedario 
+por un número dado, de esta forma, si el numero dado es 3, la letra ‘a’ 
+pasaria a ser ‘d’. Si se evaluara la letra ‘z’ esta pasaria a ser ‘c’ 
+(z -> a -> b -> c).  Escriba un programa que dado un string y un numero, 
+imprima el texto cifrado (notese que solo debe cifrar las letras, los 
+caracteres especiales y numeros quedan igual)
+*/
+
+int main (){
+	char str1[100], str2[100];
+	int i = 0, j = 0, num;
+	
+	printf("Ingrese el texto a encriptar:\n");
+	fgets(str1, sizeof(str1), stdin);
+	
+	printf("Ingrese el numero:\n");
+	scanf("%d", &num);
+	
+	for (; i <= strlen(str1); i++){
+		if (str1[i] >= 97 && str1[i] <= 122){
```

Comment:
esto solo considera letras minusculas, agrega las mayusculas de este modo:

```suggestion
		if ((str1[i] >= 65 && str1[i] <= 90) || (str1[i] >= 97 && str1[i] <= 122)){
```